mlxtend/regressor/linear_regression.py

```diff
@@ -21,13 +21,18 @@ class LinearRegression(_BaseModel, _IterativeModel, _Regressor):
     ------------
     eta : float (default: 0.01)
         solver rate (between 0.0 and 1.0)
+    method : string (default: 'direct')
+        Which method do you want to use to solve it.
+        For Gradient Descent, use 'sgd', for QR decomposition method, use
+        'qr', for SVD method, use 'svd'and for direct analytical method,
+        use 'direct'.
     epochs : int (default: 50)
         Passes over the training dataset.
         Prior to each epoch, the dataset is shuffled
         if `minibatches > 1` to prevent cycles in stochastic gradient descent.
     minibatches : int (default: None)
         The number of minibatches for gradient-based optimization.
-        If None: Normal Equations (closed-form solution)
+        If None: Direct method, QR or SVD method
```

Comment:
I suggest rewording

> If None: Direct method, QR or SVD method

to 

> If None: Direct method, QR, or SVD method (see `method` parameter for details).

Reply:
Done

Reply:
Probably same as above, I guess you used the "Done" for yourself to keep track. No worries in that case.